x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/get_signals_template.ts

```diff
@@ -45,6 +46,13 @@ export const getSignalsTemplate = (index: string) => {
       properties: {
         ...ecsMapping.mappings.properties,
         signal: signalsMapping.mappings.properties.signal,
+        threat: {
+          ...ecsMapping.mappings.properties.threat,
+          properties: {
+            ...ecsMapping.mappings.properties.threat.properties,
+            ...indicatorMapping,
```

Comment:
I don't think this is correct; we want these to be part of the nested `threat.indicator` objects, e.g. `threat.indicator[].event.*`, and this looks to generate `threat.event.*` mappings.

Reply:
hey thanks for the catch